servers/apps/matterbridge/README.md

```diff
@@ -0,0 +1,50 @@
+# Instructions for bridging if topic as group is used:
+
+Prerequisites:
+
+- Access to falcon server of FOSS United
```

Comment:
not sure where the falcon server is - is this on linode?

Reply:
yes. 172.105.55.120. It hosts Vaultwarden and wisharya.com as well